Library/Homebrew/cmd/commands.rb

```diff
@@ -7,31 +7,52 @@
 module Homebrew
   module_function
 
-  def commands
-    if ARGV.include? "--quiet"
-      cmds = internal_commands + external_commands
-      cmds += internal_developer_commands
-      cmds += HOMEBREW_INTERNAL_COMMAND_ALIASES.keys if ARGV.include? "--include-aliases"
-      puts Formatter.columns(cmds.sort)
-    else
-      # Find commands in Homebrew/cmd
-      puts "Built-in commands"
-      puts Formatter.columns(internal_commands)
-
-      # Find commands in Homebrew/dev-cmd
-      puts
-      puts "Built-in developer commands"
-      puts Formatter.columns(internal_developer_commands)
-
-      # Find commands in the path
-      unless (exts = external_commands).empty?
+  # `@commands_command` stores the code block passed into `define_command` DSL
+  @commands_command = Command.define_command do
```

Comment:
The `@commands_command =` should be set by `Command.define_command` rather than manually like this.